cmd/sonobuoy/app/gen.go

```diff
@@ -20,14 +20,14 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/vmware-tanzu/sonobuoy/pkg/client"
-	"github.com/vmware-tanzu/sonobuoy/pkg/config"
-	"github.com/vmware-tanzu/sonobuoy/pkg/errlog"
-	imagepkg "github.com/vmware-tanzu/sonobuoy/pkg/image"
 	"github.com/imdario/mergo"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	"github.com/vmware-tanzu/sonobuoy/pkg/client"
+	"github.com/vmware-tanzu/sonobuoy/pkg/config"
+	"github.com/vmware-tanzu/sonobuoy/pkg/errlog"
+	imagepkg "github.com/vmware-tanzu/sonobuoy/pkg/image"
```

Comment:
nit: it was already a problem before this PR but yours is touching it now; ideally the vmware-tanzu/sonobuoy imports would be in their own block (2nd block) and the external ones like cobra/pflag would be in a 3rd block (with stdlib being the first block)